### PR TITLE
15 add coverage GitHub action jupyter cli tests

### PR DESCRIPTION
**Pull request by plutasnyy**

### CODE REVIEW

File: tutorials/sst/tests/test_cli_commands.py [hunk 1]
```diff
@@ -6,41 +6,41 @@
...
+    expected_output_path = Path(str(outfile_path) + expected_extension)
 
+    result = cli_runner_instance.invoke(cli, ['--source', example_input, "--output", outfile_path, "--type", type])
+    print(result.output)
     assert result.exit_code == 0
-    assert os.path.exists(test_output_with_extension)
+    assert os.path.exists(expected_output_path)
+
 
-    remove_files_if_present([test_output_with_extension])
```
Comment (Adamage) [inline tutorials/sst/tests/test_cli_commands.py:34]: what happened to the cleanup now?

File: tutorials/sst/sst.py [hunk 2]
```diff
@@ -37,9 +37,9 @@ def cli(source: Path, output: Path, type: str, execute: bool) -> None:
 
     filename = construct_output_filename(outputname=output, extension=exporter.file_extension, input_name=source)
 
-    with open(filename, "w") as fpout:
-        fpout.write(output_content)
+    filename.parent.mkdir(parents=True, exist_ok=True)
```
Comment (Adamage) [inline tutorials/sst/sst.py:40]: is there any context manager I dont see here, somewhere underneath? is the file handle closed?

File: tutorials/sst/tests/test_cli_commands.py (hunk 1 above)
Comment (plutasnyy) [reply tutorials/sst/tests/test_cli_commands.py:34]: `temp_dir` is pytest fixture, which creates a temporary directory separately for each test ;)

File: tutorials/sst/sst.py (hunk 2 above)
Comment (plutasnyy) [reply tutorials/sst/sst.py:40]: According to the documentation, this is done automatically

https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text

> The file is opened and then closed.

File: tutorials/sst/tests/test_cli_commands.py (hunk 1 above)
Comment (plutasnyy) [reply tutorials/sst/tests/test_cli_commands.py:34]: https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture

**Review by Adamage — APPROVED** [review]

**Pull request merged**